pom.xml

```diff
@@ -1006,6 +1010,10 @@
               <goal>integration-test</goal>
               <goal>verify</goal>
             </goals>
+            <configuration>
+              <excludeGroups>${accumulo.integrationTestsExcludedGroups}</excludeGroups>
+              <groups>${accumulo.integrationTestsGroups}</groups>
+            </configuration>
```

Comment:
> Since you've added this indirection for failsafe, would it make sense to also add this to surefire

Presently we don't have any groups for tests we run through surefire though.

> Unfortunately, surefire and failsafe use the same user properties for these

I think this is also a good reason to have the profiles.